Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -154,19 +154,6 @@ public extension Sequence {
         return singleElement
     }
 
-    /// SwifterSwift: Remove duplicate elements based on condition.
```

Comment:
I'm not sure we can remove this, I mean ... that's the non-mutating version of that function and also is the most generic, if there's is indeed a dupe, you probably should remove the one in Array :)

Reply:
So should I return it back?

Reply:
Yes, please add it back, it should be in `Sequence` as opposed to `Array.

Reply:
fixed